Zemismart/__init__.py

```diff
@@ -98,36 +100,47 @@ def connect(self):
             self.disconnect()
             raise
 
-
     def handleNotification(self, handle, data):
         if handle == self.datahandle.getHandle() and data[0] == self.response_start_byte:
             if data[1] == self.get_battery_cmd[0]:
                 battery = data[7]
-                print("Got battery: " + str(battery))
                 self.save_battery(battery)
+                self.last_command_status = True
             elif data[1] == self.get_position_cmd[0]:
                 pos = data[5]
-                print("Got position: " + str(pos))
                 self.save_position(pos)
+                self.last_command_status = True
             elif data[1] == self.finished_moving_cmd[0]:
                 pos = data[4]
-                print("Finished moving to position: " + str(pos))
                 self.save_position(pos)
-    
+                self.last_command_status = True
+            elif data[1] == 0x41:  # notify position

```

Comment:
To keep things consistent, please define this command (`0x41`) like `get_battery_cmd`, `get_position_cmd` etc. has been defined.